microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -305,6 +322,15 @@ When these enterprise features are on, the browser sends auditing and diagnostic
 Please contact your enterprise administrator to understand your company's policies for such data collection.
 
 
+<!-- ====================================================================== -->
+## Entity extraction
+
+Every 24 hours, Microsoft Edge downloads a list of supported sites that have special entity extraction templates.  The templates are specific to each website.  When you create a new item in your collection, Microsoft Edge verifies that the site you are collecting from is in the list of supported sites.  If the site is in the list, Microsoft Edge pings the entity extraction service for the template of the specific site.  No user identifiers are associated with the request to the service.  The template attempts to identify the name, price, ratings, primary image, and other data about the item being collected.  If the site you are collecting from is not on the supported list site, Microsoft Edge does not download a template.  Templates allow collection items to be created locally on the device.  No data about the collection items are sent to the service when creating the collection.
```

Comment:
@yuanjunz - need E E team to review.  "collection" occurs 8x in this paragraph.

Paragraph needs to be split up for readability.

Reply:
Simplify below:
Microsoft Edge uses entity extraction templates specific to a list of supported websites to identify the name, price, ratings, primary image, and other data about the item being collected locally.  When creating the collection, no user identifiers or other data are sent to Microsoft services.
You can delete templates stored on the device by going to `edge://settings/clearBrowserData`, selecting the desired time range and type of data, and then selecting the **Clear now** button.

Reply:
done.
entered [fresh Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2778/files#r1300819539)